src/py/flwr/client/app.py

```diff
@@ -222,7 +258,8 @@ class `flwr.client.Client`.
 def start_numpy_client(
     *,
     server_address: str,
-    client: NumPyClient,
+    client_fn: Optional[Callable[[str], NumPyClient]] = None,
```

Comment:
```suggestion
    client_fn: Optional[Callable[[str], ClientLike]] = None,
```

Reply:
But this should be `NumPyClient` right?

Reply:
The `ClientFn` returns a `ClientLike` object (i.e., either a `Client` or a `NumPyClient`)